llvm/tools/llvm-exegesis/lib/Target.h

```diff
@@ -32,6 +32,21 @@
 #include "llvm/TargetParser/SubtargetFeature.h"
 #include "llvm/TargetParser/Triple.h"
 
+// Before kernel 4.17, Linux did not support MAP_FIXED_NOREPLACE, so if it is
```

Comment:
We should not be relying on the fact that BenchmarkRunner.cpp includes Target.h. Let's move these definitions to a separate file (e.g. MMap.h) and include that file in files that use these symbols.

Reply:
Good point. Moved over to `MmapUtils.h`.